gateway-ha/src/main/java/io/trino/gateway/ha/security/AuthorizationManager.java

```diff
@@ -35,9 +39,42 @@ public AuthorizationManager(AuthorizationConfiguration configuration,
         else {
             lbLdapClient = null;
         }
+        this.authorizationConfiguration = configuration;
     }
 
-    public Optional<String> getPrivileges(String username)
+    @VisibleForTesting
+    public AuthorizationManager(Map<String, UserConfiguration> presetUsers, LbLdapClient lbLdapClient, AuthorizationConfiguration authorizationConfiguration)
+    {
+        this.presetUsers = presetUsers;
+        this.lbLdapClient = lbLdapClient;
+        this.authorizationConfiguration = authorizationConfiguration;
+    }
+
+    public String getPrivileges(String username)
+    {
+        if (authorizationConfiguration == null) {
+            return "ADMIN_USER_API";
```

Comment:
I’d prefer not to give any privileges when `authorizationConfiguration` is not set.